magefiles/magefile.go

```diff
@@ -316,7 +316,7 @@ func (ci CI) UnregisterSprayproxy() {
 }
 
 func RunE2ETests() error {
-	labelFilter := utils.GetEnv("E2E_TEST_SUITE_LABEL", "!upgrade-create && !upgrade-verify && !upgrade-cleanup && !release-pipelines")
+	labelFilter := utils.GetEnv("E2E_TEST_SUITE_LABEL", "!upgrade-create && !upgrade-verify && !upgrade-cleanup && release-pipelines")
```

Comment:
This is exactly the change I was talking about on https://github.com/konflux-ci/e2e-tests/pull/1278. I thought with https://issues.redhat.com/browse/KONFLUX-3383 we wouldn't have to do this anymore to have the `release-pipelines` suite run on changes to it in e2e-tests PRs

Reply:
There was some issue in the test selection logic when I pushed the commit last time. Removed the change this time.

Reply:
Sure. I am still looking for someone to definitively tell me that with KONFLUX-3383 closed, PRs modifying a file in the release-pipelines suite causes the release-pipelines suite to run on the PR without needing a magefile change. I see you now removed the magefile change, but I don't see the release-pipelines suite run in the checks

Reply:
Under the current test cases selection mechanism, if only some test files are changed in release-pipelines,  the test cases in these test files will be run, not the whole release-pipelines suite.  If the common file there, such as, const.go, the test suite will be ran. Is that ok ?

Reply:
Yeah that sounds great